### ISSUE TITLE
QUESTION: How can I see all databases in the MSSQL server?

### ISSUE THREAD

**Issue by luisxkimo** [dbgate/dbgate#236]
**Details:**
I see that with the last versions of DB Gate, its needed new environment variable with default database name: DATABASE_connName.

For example, if I use "master" as default, then in the list of databases I cannot see the other created databases in the server, but exists because I can run queries on the other databases.

**Version Information [might be relevant to your issue]**
![Screenshot 2022-02-28 091806](https://user-images.githubusercontent.com/6990299/155952524-236d50be-b545-4e03-94a0-9922b0c21d46.png)

**Screenshot [if appropriate]**:
![Screenshot 2022-02-28 094756](https://user-images.githubusercontent.com/6990299/155952537-c4d50c64-8573-42a4-984f-6f27a6a4e6db.png)

**Additional info**
Nothing

**Comment by janproch** [comment]
It is not neccessary to specify database name with DATABASE_ environment variable. When you omit this variable, you will see all databases in DbGate interface.

**Comment by luisxkimo** [comment]
Hi @janproch ,

Thanks for the quick reply. The thing is when we not use this variable, we se an error on the web:
![image](https://user-images.githubusercontent.com/6990299/155974064-f1e868ab-f232-4331-bc00-b50a3693ea39.png)

And if we put the mouse over it says: "The config.options.database must be a type of string"

Here is the log of the container:

```json
Using connections from ENV variables:
[
  {
    "_id": "***",
    "engine": "mssql@dbgate-plugin-mssql",
    "server": "***",
    "user": "***",
    "password": "***",
    "port": "1433",
    "useDatabaseUrl": false,
    "defaultDatabase": null,
    "singleDatabase": false,
    "displayName": "***"
  }
]
Calling init controller for controller /connections
Creating directory /root/dbgate-data
Calling init controller for controller /config
Calling init controller for controller /scheduler
Creating directory /root/dbgate-data/files
Creating directory /root/dbgate-data/run
DbGate API listening on port 3000
....
```

![image](https://user-images.githubusercontent.com/6990299/155974970-339296bf-95a8-4cd7-aa21-edc9cbaa0537.png)

**Comment by janproch** [comment]
Intereseting, I am testing it now and it works.

Did it work in past? The only change in last version is using debian docker image instead of alpine, because of SQLite. But I tested it on both images, and it works.

Does it work with alpine image?
```image: dbgate/dbgate:alpine```

**Comment by janproch** [comment]
OK, I replicated it just now

**Comment by luisxkimo** [comment]
Tested right now with **dbgate:alpine** and still not works

**Comment by janproch** [comment]
It should be fixed in newest BETA (few minutes old), in both alkine and debian image. I have replicated it with last version (it occured only when MSSQL run also in docker container, when connecting to MSSQL running under windows it worked)

Sorry for confustion.

**Comment by janproch** [comment]
Fixed in 4.7.1

Thanks for reporting

**Comment by luisxkimo** [comment]
Thanks so much for the quick fix @janproch
